### PR TITLE
feature/SKED-3980: Update Node from 9.x to 10.x

### PR DESCRIPTION
**Pull request by gevalo1**
See JIRA: https://skedify.atlassian.net/browse/SKED-3980

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,7 +5,8 @@ cache:
 notifications:
   email: false
 node_js:
-  - '9'
+  - "10"
+  - "9"
```
Comment (RobinMalfait) [inline .travis.yml]: Do we still need node 9? I think this will make our builds slower and I think we can drop it if we have everything on node 10.

**Comment by RobinMalfait** [comment]
package-lock.json should use the npm registry instead of the team registry.

File: .travis.yml [hunk 2]
```diff
@@ -5,9 +5,13 @@ cache:
...
 node_js:
-  - '9'
+  - "10"
+before_script:
+  - npm run build
+  - npm audit --only=prod --audit-level=critical
 after_success:
-  - ./scripts/release.sh
+  - npm run publish-coverage
+  - npm run travis-deploy-once "npm run semantic-release"
```
Comment (RobinMalfait) [inline .travis.yml]: Where/how do we check that it is the master branch?

Comment (gevalo1) [reply .travis.yml]: https://github.com/semantic-release/semantic-release/blob/caribou/docs/usage/configuration.md#branch

`master` is the default release branch.

**Pull request merged**